A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -1004,6 +1042,22 @@ var A320_Neo_UpperECAM;
                         important: true,
                         isActive: () => this.showLandingInhibit,
                     },
+                    {
+                        message: "AP OFF",
+                        style: "fail-3",
+                        important: true,
+                        isActive: () => {
+                            return (this.getAPWarning(3));
```

Comment:
Don't call functions that change the state or SimVars in isActive. IsActive may be called multiple times at the same time in future.